patches/minecraft/net/minecraft/server/network/ServerGamePacketListenerImpl.java.patch

```diff
@@ -64,11 +64,16 @@
 +                  if(!ServerGamePacketListenerImpl.this.f_9743_.canInteractWith(entity, 1.5D)) return; //Forge: If the entity cannot be reached, do nothing. Original check was dist < 6, range is 4.5, so vanilla used padding=1.5
                    ItemStack itemstack = ServerGamePacketListenerImpl.this.f_9743_.m_21120_(p_143679_).m_41777_();
                    InteractionResult interactionresult = p_143680_.m_143694_(ServerGamePacketListenerImpl.this.f_9743_, entity, p_143679_);
-+                  if (net.minecraftforge.common.ForgeHooks.onInteractEntityAt(f_9743_, entity, entity.m_20182_(), p_143679_) != null) return;
                    if (interactionresult.m_19077_()) {
-                      CriteriaTriggers.f_10565_.m_61494_(ServerGamePacketListenerImpl.this.f_9743_, itemstack, entity);
-                      if (interactionresult.m_19080_()) {
-@@ -1243,7 +_,7 @@
+@@ -1237,13 +_,15 @@
+ 
+                public void m_142143_(InteractionHand p_143682_, Vec3 p_143683_) {
+                   this.m_143678_(p_143682_, (p_143686_, p_143687_, p_143688_) -> {
++                     InteractionResult onInteractEntityAtResult = net.minecraftforge.common.ForgeHooks.onInteractEntityAt(p_143686_, p_143687_, p_143683_, p_143682_);
```

Comment:
Hmm, it's functionally equivalent but not semantically. It should either be changed to what was in the original PR or the 1.19.x version should be updated to use the lambda parameters.

https://github.com/MinecraftForge/MinecraftForge/blob/c8f9e59fef4c52875601014c9fa18e41be619d0f/patches/minecraft/net/minecraft/server/network/ServerGamePacketListenerImpl.java.patch#L82-L88

Reply:
Came up with what was made in the original PR